keras_cv/layers/preprocessing/solarization.py

```diff
@@ -93,41 +93,51 @@ def __init__(
         )
         self.value_range = value_range
 
-    def get_random_transformation(self, **kwargs):
+    def get_random_transformation_batch(self, batch_size, **kwargs):
         return (
-            self.addition_factor(dtype=self.compute_dtype),
-            self.threshold_factor(dtype=self.compute_dtype),
+            self.addition_factor(
```

Comment:
pretty tiny nit, but can we change the random transformation format from:

```
(addition, threshold)
```

to

```
{
"addition": addition,
"threshold": threshold
}
```
?

The reason I think this is better is that we are seeing some usage of the get_transformation() APIs directly.

Reply:
@LukeWood done. Changed `get_random_transformation_batch` to return a dictionary.